src/App.js

```diff
@@ -1,9 +1,12 @@
 import './App.css';
+import Header from './components/Header';
+import About from './components/about/About';
```

Comment:
Looking at the official recommendations: https://reactjs.org/docs/faq-structure.html

I guess it should be not be that nested? So either:

```suggestion
import Header from './header/Header';
import About from './about/About';
```
or:
```suggestion
import Header from './components/Header';
import About from './components/About';
```

What do you think?

Reply:
Agreed. Thanks for the suggestion. Too much nesting is definitely not good in the long run.